composer.json

```diff
@@ -14,14 +14,26 @@
       "email": "josh@findsomehelp.com"
     }
   ],
+  "require": {
+    "php": ">=5.4.0"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "~4.0||~5.0",
+    "scrutinizer/ocular": "~1.1",
```

Comment:
Ocular and Code Sniffer isn't actually necessary for this repo.  I tossed it in since it's super handy.   it can be removed if ya want.  It doesn't look like this repo actually sticks to its code style requirements.